myapp/models.py

```diff
@@ -137,8 +141,14 @@ class Comment(models.Model):
     def __str__(self):
         return f"{self.author}'s comment for '{self.liked_object}': {self.comment[:15]}"
 
+    def save(self, **kwargs):
+        if not self.id:
+            self.date = timezone.now() - timedelta(days=365)
+        super().save(**kwargs)
+
     class Meta:
         verbose_name = '3_Comment'
+        ordering = ["-id"]
```

Comment:
What is the reason for this ordering? Or it's just for a try?)

Reply:
only for more convenient tracking changes in the admin panel (no needed scroll after F5)